main.tex

```diff
@@ -599,22 +602,60 @@
 \end{enumerate}
 
 \item  USE OF COMMON PROPERTY
+\newline These rules entirely replace section 4 ("Use of common property") in the model regulations.
 
 \begin{enumerate}[label=\arabic{enumi}.\arabic*.]
 
 \item  General
 
 \begin{enumerate}[label=(\arabic*)]
 
-\item  An Owner or Occupier of a Lot must not obstruct or prevent the lawful use and enjoyment of the Common Property by any other person entitled to use the Common Property, or use the Common Property, or allow the Common Property to be used by any other person in such a manner as to unreasonably interfere with the quiet enjoyment of any person entitled to use the Common Property.
```

Comment:
This is the most impartant rule of the document, and yet it's so outrageously vague.